include/zephyr/drivers/gpio.h

```diff
@@ -513,7 +513,8 @@ enum gpio_int_mode {
 
 enum gpio_int_trig {
 	/* Trigger detection when input state is (or transitions to)
-	 * physical low. (Edge Failing or Active Low) */
+	 * physical low. (Edge Falling or Active Low)
```

Comment:
LGTM - can we eliminate the whitespace change?

Reply:
No, compliance checks will yell at me if I do that.